### PR TITLE
Added kns function

### PR DESCRIPTION
**Pull request by rodrigovalin**

### CODE REVIEW

File: functions.sh [hunk 1]
```diff
@@ -189,6 +189,21 @@ choose_all(){
     fi
 }
 
+# Changes the default Namespace, requires the `ns` plugin.
```
Comment (bznein) [inline functions.sh]: [nit] we could avoid the need for `ns` by using `kubectl config set-context --current --namespace=`

**Review by bznein — APPROVED** [review]
LGTM thanks! One minor comment

**Comment by bznein** [comment]
Also please make sure Shellcheck is happy :P

**Comment by rodrigovalin** [comment]
Done, I need approval to run the tests please

**Comment by rodrigovalin** [comment]
Can't merge myself @bznein :(

**Comment by bznein** [comment]
Pfff I need to check the settings of the repo :D

**Pull request merged**